v2/config/rbac/role.yaml

```diff
@@ -65,11 +64,17 @@ rules:
   resources:
   - configmaps
   - pods
-  - secrets
   verbs:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - secrets
+  verbs:
+  - '*'
```

Comment:
Should probably be specific with the verbs as well. If we can name the secrets as well the better.

Reply:
fixed, forgot to add the file after the make generate

I will look into scoping to name as well, which will depend on if I can get fields.AndSelectors to work for the cache selector, which will be part of 24286

For now, the goal was to accomplish what was set out in the pr description